living/komi-zyrian.md

```diff
@@ -7,13 +7,19 @@ News:
 - http://komikerka.ru/
 - https://komiinform.ru/news/e/161
 - https://www.nbrkomi.ru/kraevedenie/vyltoryas (accessible only with Russian IP)
+- https://rkomi.ru/language/kv (accessible only with Russian IP)
+- https://komivoityr.ru/news
+- 
 
 Culture / History:
 - https://komikyv.org/
+- https://cultmap.nbrkomi.ru/km (accessible only with Russian IP)
 - 
 
 Government:
-- 
+- https://gov.rkomi.ru/kv (accessible only with Russian IP)
+- https://komi.gsrk.ru (accessible only with Russian IP)
+- https://adm.rkomi.ru/language/kv (accessible only with Russian IP)
```

Comment:
Our crawl is run from the US East region so this will not be of benefit there, but approving this addition anyway since this may be of a benefit to others.